neuralmonkey/readers/plain_text_reader.py

```diff
@@ -60,6 +60,7 @@ def t2t_tokenized_text_reader(encoding: str = "utf-8") -> PlainTextFileReader:
     def reader(files: List[str]) -> Iterable[List[str]]:
         lines = string_reader(encoding)
         for line in lines(files):
+            line = line.strip()
```

Comment:
Tohle tu nemá bejt, viz dokumentace T2T tokenizéru:
https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/data_generators/tokenizer.py#L35

Reply:
To je sice pekne, ze to tam nema byt, ale porovnej si cteni vstupu z t2t a u nas a pak jejich a nasi tokenizaci a uvidis, ze se jim tam z nejakeho duvodu newliny na konci radku "ztrati", kdezto u nas ne. Pak ti samozrejme vyjdou odlisne vstupy, coz se samozrejme veze dal.

Ztratil jsem tim skoro cele odpoledne nez jsem si tohodle vsiml.

Reply:
Správně by asi bylo
```
line = line.rstrip("\n")
```